src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -295,6 +362,17 @@ export default class Fms {
     }
 
     /**
+     * Return an array of waypoints in the flight plan that have altitude restrictions
+     *
+     * @for Fms
+     * @method getAltitudeRestrictedWaypoints
+     * @return {array<WaypointModel>}
+     */
+    getAltitudeRestrictedWaypoints() {
```

Comment:
and here we are doing the same type of thing as the getters I commented on above.  This way feels much more correct.